### ISSUE TITLE
tests: integration

### ISSUE THREAD

**Issue by pared** [iterative/dvc-render#25]
We need some end-to-end tests verifying that provided renderers gets converter to expected HTML. So far we only have those for parallel coordinates plot.

**Comment by daavoo** [comment]
Duplicate https://github.com/iterative/dvc-render/issues/4 ?

**Comment by pared** [comment]
Not quite, those two are about testing HTML, so verifying that the rendering gets done and no errors are thrown on JS side. However what I mean here is that we should prepare a tests making sure that particular data points get converted into particular HTML. They might be joined with #4 but not necessarily. Our current tests resemble unit tests more than end-to-end tests.
